src/components/twilio-voice-basic-call-control/public/twilio-voice-basic-call-control.js

```diff
@@ -85,6 +85,24 @@ class TwilioVoiceBasicCallControl extends HTMLElement {
     this.#showElement(`#${callSid}-unmute`, shouldMute);
   }
 
+  async #handleRemoveParticipant(callSid) {
+    const response = await await fetch(
+      `/twilio-voice-basic-call-control/conferences/${
+        this.#conferenceSid
+      }/participants/${callSid}`,
+      {
+        headers: {
+          'Content-Type': 'application/json',
+        },
```

Comment:
We're not sending any content so I don't think we need this. It's just the HTTP METHOD DELETE right?

Reply:
yes just DELETE